configs/secrets/dev-genesis-secrets.js

```diff
@@ -8,11 +8,22 @@ const path = require('path');
 config['veres-one'].adminPassphrase =
   'password';
 config['veres-one'].peers = [];
+
+// maintainer
 config['veres-one'].maintainerConfigFile =
   path.join(config.paths.secrets, 'maintainer.ddo');
-config['veres-one'].governorsConfigFile =
-  path.join(config.paths.secrets, 'governors.ddo');
 config['veres-one'].maintainerPassphrase =
   'insecure_eyium0phookoh9geeshewomaekuoTeib';
+
+// governors
+config['veres-one'].governorsConfigFile =
+  path.join(config.paths.secrets, 'governors.ddo');
```

Comment:
`ddo` isn't a thing anymore, probably shouldn't name it that. Perhaps just `.json` or `.jsonld`?

Reply:
Changed.